buildSrc/gradle.properties

```diff
@@ -1,7 +1,17 @@
 jacksonVersion = 2.13.3
 unirestVersion=3.13.10
-cordaApiVersion=5.0.0.505-Beta1.0-HC00
 
-# R3 internal repository
-artifactoryContextUrl=https://staging.download.corda.net/maven/20ede3c6-29c0-11ed-966d-b7c36748b9f6-Beta1.0-HC00
+cordaApiVersion=5.0.0.522-Fox1.0-RC01
+#cordaApiVersion=5.0.0.519-Beta-Fox-HC03
+#cordaApiVersion=5.0.0.514-beta-1670584271886
+#cordaApiVersion=5.0.0.508-beta+
+#cordaApiVersion=5.0.0.507-beta-1669967318895
+#cordaApiVersion=5.wrong
```

Comment:
I understand this is for testing purposes but should it be committed to release branch , same with other values here ?

Reply:
Agreed it needs tidying up